canopen/objectdictionary/eds.py

```diff
@@ -2,9 +2,9 @@
 import io
 import logging
 try:
-    from configparser import ConfigParser
+    from configparser import ConfigParser, NoOptionError
 except ImportError:
-    from ConfigParser import RawConfigParser as ConfigParser
+    from ConfigParser import RawConfigParser as ConfigParser, NoOptionError
```

Comment:
This does not work. Maybe put on a new line?

Reply:
It seems the problem was not in this line, instead there were some non-ASCII quotes in the text that I had copied from the DS306 document. I adjusted them to ASCII quotes and now it seems fine.